### DISCUSSION TITLE
Link to "Request a Record for Review" not working correctly when deploying to Vercel

### DISCUSSION THREAD

**Discussion by MacCode7110** [vercel/next.js#27349] [Help]
### What version of Next.js are you using?

10.1.1

### What version of Node.js are you using?

12.0.0

### What browser are you using?

Internet Explorer

### What operating system are you using?

Windows 10

### How are you deploying your application?

Vercel

### Describe the Bug

On localhost, the link to "Request a Record for Review" HTML page works correctly, although there is an internal server error because I am relying on using a backend platform called FormSubmit to handle emailing forms to me, and linking an action to that platform for the HTML form code only works when you have a deployment/official URL. 
When deploying to Vercel, I get 404 Page Not Found when I click the link to the "Request a Record for Review" HTML page. I am not sure if I linked to the HTML page correctly (I used a relative path), and I am not sure if this is causing the 404 Page Not Found issue. 

### Expected Behavior

I expect the link to the "Request a Record for Review" HTML page to work correctly, and the page should actually should show the form since I linked to the FormSubmit backend platform correctly, and now that my website is deployed to Vercel.

### To Reproduce

![NuMetalPowerZoneProductionScreenshot2](https://user-images.githubusercontent.com/67719286/126254270-0fd570b2-8748-45da-8f1f-baa42d230e6c.PNG)

Here is a link to my GitHub page: 

https://github.com/MacCode7110/TheNuMetalPowerZone

**Comment by ijjk** [comment]
Hi, it looks like you are adding HTML files in your `pages` directory which won't be served automatically. If you want these HTML to be accessible you can add them in your `public` folder instead.
